onnxruntime/core/optimizer/constant_folding.cc

```diff
@@ -196,10 +196,10 @@ Status ConstantFolding::ApplyImpl(Graph& graph, bool& modified, int graph_level,
       ORT_ENFORCE(fetches.size() == node->OutputDefs().size());
       converted_to_constant = true;
       for (size_t fetch_idx = 0; fetch_idx < fetches.size(); ++fetch_idx) {
-        OrtValue& ort_value = fetches[fetch_idx];
+        auto* constant_arg_out = node->MutableOutputDefs()[fetch_idx];
         // XXX: Add support for SparseTensors outputs when we have sparse outputs
-        if (!ort_value.IsTensor()) {
-          LOGS(logger, WARNING) << "Unsupported output type of " << ort_value.Type()
+        if (!utils::HasTensorType(*constant_arg_out->TypeAsProto())) {
+          LOGS(logger, INFO) << "Unsupported output type of " << constant_arg_out->Type()
```

Comment:
Changed WARNING to INFO in order to make this less jarring as it came up in another issue